Library/Homebrew/dev-cmd/bump-formula-pr.rb

```diff
@@ -259,8 +262,10 @@ def bump_formula_pr
         remote = Utils.popen_read("hub fork 2>&1")[/fatal: remote (.+) already exists\./, 1]
         odie "cannot get remote from 'hub'!" if remote.to_s.empty?
         safe_system "git", "push", "--set-upstream", remote, "#{branch}:#{branch}"
-        safe_system "hub", "pull-request", "--browse", "-m",
-          "#{formula.name} #{new_formula_version}#{devel_message}\n\nCreated with `brew bump-formula-pr`."
+        pr_message = "#{formula.name} #{new_formula_version}#{devel_message}\n\nCreated with `brew bump-formula-pr`."
+        user_message = ARGV.value('message')
+        pr_message += "\n\n---\n\n#{user_message}" if user_message
```

Comment:
Use `EOS.undent` here to make it more obvious how the newlines fit in. I realise `\n` is used above but it'd be nice to improve that too.